projects/packages/my-jetpack/_inc/components/add-license-screen/index.jsx

```diff
@@ -19,14 +18,16 @@ import GoBackLink from '../go-back-link';
  * @returns {object} The AddLicenseScreen component.
  */
 export default function AddLicenseScreen() {
-	useEffect( () => {
-		const { apiRoot, apiNonce } = window?.myJetpackRest || {};
-		restApi.setApiRoot( apiRoot );
-		restApi.setApiNonce( apiNonce );
-	}, [] );
-
 	const { recordEvent } = useAnalytics();
-	const { availableLicenses, fetchingAvailableLicenses } = useAvailableLicenses();
+	const { data: availableLicenses = [], isLoading: fetchingAvailableLicenses } = useJetpackApiQuery(
```

Comment:
I saw that in #36028 we used variable names like `is{entityName}Loading`, I'd say we can do the same here for consistency.